src/Microsoft.ML.Data/Transforms/Normalizer.cs

```diff
@@ -384,7 +384,11 @@ public ColumnFunctionAccessor(ImmutableArray<ColumnOptions> infos)
         [BestFriend]
         internal readonly IReadOnlyList<IColumnFunction> ColumnFunctions;
 
+        /// <summary>
+        /// The configuration of the normalizer. The i-th element describes the i-th input-output column pair.
+        /// </summary>
         public readonly ImmutableArray<ColumnOptions> Columns;
```

Comment:
>ColumnOptions [](start = 39, length = 13)

Since you are doing the Normalizer scrubbing, we need to rename `ColumnOptions` here. It really does not make sense for it to be named like this.
[https://github.com/dotnet/machinelearning/issues/2854](https://github.com/dotnet/machinelearning/issues/2854)

Reply:
It doesn't look like that thread converges already..

---
In reply to: [265349922](https://github.com/dotnet/machinelearning/pull/2888#discussion_r265349922) [](ancestors = 265349922)